src/thermo/MixtureFugacityTP.cpp

```diff
@@ -44,6 +47,25 @@ int MixtureFugacityTP::reportSolnBranchActual() const
     return iState_;
 }
 
+// ---- Molar Thermodynamic Properties ---------------------------
+double MixtureFugacityTP::enthalpy_mole() const
+{
+   _updateReferenceStateThermo();
+   double h_ideal = RT() * mean_X(m_h0_RT);
+   double h_nonideal = hresid();
+   return h_ideal + h_nonideal;
+}
+
+
+double MixtureFugacityTP::entropy_mole() const
+{
+    _updateReferenceStateThermo();
+    double sr_ideal = GasConstant * (mean_X(m_s0_R) - sum_xlogx()
+                                                - std::log(pressure()/refPressure()));
```

Comment:
The indentation here is a little odd. Also, what does "sr" mean?

Reply:
The naming convention is carried forward from RedlichKwong file. I have now changed it to s_ideal and s_nonideal, where s stands for the entropy.